### ISSUE TITLE
object not found

### ISSUE THREAD

**Issue by davidvilanova** [sahirbhatnagar/manhattanly#11]
Hi ,
I have a problem to create the following Volcano plot.

```
 colnames(tab)
[1] "logFC" "Pval"  "taxa" 
```

```
volc <- volcanor(tab,p="Pval",effect_size="logFC",snp="taxa",gene="taxa")
volcanoly(volc)
Error in eval(expr, envir, enclos) : object 'logFC' not found

```

```
str(volc)
List of 8
 $ data           :'data.frame':	387 obs. of  5 variables:
  ..$ EFFECTSIZE: num [1:387] 0 0 0 0.937 0.83 ...
  ..$ P         : num [1:387] 0 0 0 0.173 0.151 ...
  ..$ taxa      : Factor w/ 387 levels "WGS_100","WGS_101",..: 208 88 378 1 5 12 370 137 21 380 ...
  ..$ taxa.1    : Factor w/ 387 levels "WGS_100","WGS_101",..: 208 88 378 1 5 12 370 137 21 380 ...
  ..$ LOG10P    : num [1:387] Inf Inf Inf 0.761 0.821 ...
 $ pName          : chr "Pval"
 $ effectName     : chr "logFC"
 $ xlabel         : chr "logFC"
 $ snpName        : chr "taxa"
 $ geneName       : chr "taxa"
 $ annotation1Name: logi NA
 $ annotation2Name: logi NA
 - attr(*, "class")= chr "volcanor"
```

**Comment by sahirbhatnagar** [comment]
I am able to reproduce this problem. thanks for reporting. I will submit a fix soon.

**Comment by sahirbhatnagar** [comment]
Sorry for the delay. This should now be fixed in the most recent release:

```r
HapMap2 <- HapMap
colnames(HapMap2) <- c("chromosome","baseposition","Pval","rs","zvalue", "logFC","taxa","dist")
p <- volcanoly(HapMap2, 
               p = "Pval",
               effect_size = "logFC",
               snp = "rs",
               gene = "taxa")
```

See the following link for the resulting plot: https://plotly.com/~sahirbhatnagar/151/

**Comment by sahirbhatnagar** [comment]
Fixed with #24
